src/Utils/api.js

```diff
@@ -28,6 +28,12 @@ export const getReviewComments = (reviewId) => {
   });
 };
 
+export const getReviewsByCategory = (category) => {
+  return boardGames.get(`reviews?category=${category}`).then((res) => {
```

Comment:
For refactoring, you could combine this with your getReviews function using axios params (see React Projects in the NC Notes for more information).